### ISSUE TITLE
Mod not loading on startup

### ISSUE THREAD

**Issue by miarn** [mhgar/EmploymentPriority#2]
Installed the mod via Thunderstore, no other mods installed. The game starts fine but the mod appears to not load. The system is running an AMD Ryzen 5 4650G, GTX970, and Windows 11. I've tested a couple other mods which appear to work fine.

Below is the output from the console:

````
[Message:   BepInEx] BepInEx 5.4.16.0 - Timberborn (24/12/2021 3:02:13 AM)
[Info   :   BepInEx] Running under Unity v2022.1.0.9261453
[Info   :   BepInEx] CLR runtime version: 4.0.30319.42000
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Bits64, Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.16.0]
[Info   :   BepInEx] 1 patcher plugin loaded
[Info   :   BepInEx] Patching [UnityEngine.CoreModule] with [BepInEx.Chainloader]
[Message:   BepInEx] Preloader finished
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 3 plugins to load
[Info   :   BepInEx] Loading [TimberAPI 0.2.0]
[Info   :   Console] Creating new asset area with prefix: timberApi
[Info   :   Console] Prefix timberApi, assets: 1
[Info   : TimberAPI] TimberAPI is loaded!
[Info   :   BepInEx] Loading [EmploymentPriority 1.0.4]
[Error  : Unity Log] MissingFieldException: Field not found: TimberbornAPI.DependencySystem.IDependencyRegistry TimberbornAPI.TimberAPI.DependecyRegistry Due to: Could not find field in class
Stack trace:
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()

[Info   :   BepInEx] Loading [MoreMapDirectories 1.0.0]
[Error  : Unity Log] AmbiguousMatchException: Ambiguous match found.
Stack trace:
System.RuntimeType.GetMethodImplCommon (System.String name, System.Int32 genericParameterCount, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Reflection.CallingConventions callConv, System.Type[] types, System.Reflection.ParameterModifier[] modifiers) (at <31c0f51ac5a24a22ba784db24f4ba023>:0)
System.RuntimeType.GetMethodImpl (System.String name, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Reflection.CallingConventions callConv, System.Type[] types, System.Reflection.ParameterModifier[] modifiers) (at <31c0f51ac5a24a22ba784db24f4ba023>:0)
System.Type.GetMethod (System.String name, System.Reflection.BindingFlags bindingAttr) (at <31c0f51ac5a24a22ba784db24f4ba023>:0)
HarmonyLib.AccessTools.DeclaredMethod (System.Type type, System.String name, System.Type[] parameters, System.Type[] generics) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchTools.GetOriginalMethod (HarmonyLib.HarmonyMethod attr) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
Rethrow as HarmonyException: Ambiguous match for HarmonyMethod[(class=Timberborn.MapSystem.MapRepository, methodname=CustomMapNameToFileName, type=Normal, args=undefined)]
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.CreateAndPatchAll (System.Reflection.Assembly assembly, System.String harmonyInstanceId) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
TBMoreMapsDirectories.MoreMapsDirectoriesPlugin.Awake () (at <a4802ee27f00483e9eaaae70a3e104c3>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()

[Message:   BepInEx] Chainloader startup complete
[Info   : Unity Log] Starting game version 0.1.1.1-db764df-sw
[Info   : Unity Log] System info:
System: Windows 11  (10.0.22000) 64bit
CPU: AMD Ryzen 5 PRO 4650G with Radeon Graphics
GPU: NVIDIA GeForce GTX 970
GPU memory: 4043MB
RAM: 14225MB
Drives:
  C:\ 285.47GB free from 658.00GB

[Info   : Unity Log] Modded: true, BepInEx
[Info   :   Console] Loading scene: Global, prefixes in scene: timberApi
[Info   :   Console] Loading scene: MainMenu, prefixes in scene:
[Info   : Unity Log] Initialized configurators for MainMenu
[Info   : Unity Log] Initialized configurators for Global
Setting breakpad minidump AppID = 1062090
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561198034393243 [API loaded no]
[Info   : Unity Log] Successfully connected to the Steam client.
[Info   : Unity Log] Reading WorldSave from compressed stream
[Info   : Unity Log] Deserialized save with version: 0.1.1.1-db764df-sw
[Info   :   Console] Unloading scene: MainMenu
[Info   :   Console] Loading scene: InGame, prefixes in scene:
[Info   : Unity Log] Initialized configurators for InGame
[Info   : Unity Log] Initialized configurators for Global
[Info   : Unity Log] Loading game from Alexandrina - 2022-01-03 10h31m, Day 24-5.autosave at 2022-01-03 10:32:18Z
[Info   : Unity Log] Reading WorldSave from compressed stream
[Info   : Unity Log] Deserialized save with version: 0.1.1.1-db764df-sw
[Info   : Unity Log] Initialized SettlementName to Alexandrina
[Info   : Unity Log] Load time: 37051 ms

````

**Comment by tehseano** [comment]
Also getting the same error, different hardware/os and mods.

```
[Message:   BepInEx] BepInEx 5.4.16.0 - Timberborn (12/24/2021 4:47:18 AM)
[Info   :   BepInEx] Running under Unity v2022.1.0.9261453
[Info   :   BepInEx] CLR runtime version: 4.0.30319.42000
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Bits64, Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.16.0]
[Info   :   BepInEx] 1 patcher plugin loaded
[Info   :   BepInEx] Patching [UnityEngine.CoreModule] with [BepInEx.Chainloader]
[Message:   BepInEx] Preloader finished
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 7 plugins to load
[Info   :   BepInEx] Loading [Beaver Buffs 1.1.0]
[Info   :Beaver Buffs] Plugin Beaver Buffs is loaded!
[Info   :   BepInEx] Loading [Unified Factions 1.1.1]
[Info   :Unified Factions] Plugin Unified Factions is loaded!
[Info   :   BepInEx] Loading [TimberAPI 0.2.0]
[Info   :   Console] Creating new asset area with prefix: timberApi
[Info   :   Console] Prefix timberApi, assets: 1
[Info   : TimberAPI] TimberAPI is loaded!
[Info   :   BepInEx] Loading [DistrictExtender 1.0.0.0]
[Info   :   BepInEx] Loading [EmploymentPriority 1.0.4]
[Error  : Unity Log] MissingFieldException: Field not found: TimberbornAPI.DependencySystem.IDependencyRegistry TimberbornAPI.TimberAPI.DependecyRegistry Due to: Could not find field in class
Stack trace:
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()

[Info   :   BepInEx] Loading [MoreMapDirectories 1.0.0]
[Error  : Unity Log] AmbiguousMatchException: Ambiguous match found.
Stack trace:
System.RuntimeType.GetMethodImplCommon (System.String name, System.Int32 genericParameterCount, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Reflection.CallingConventions callConv, System.Type[] types, System.Reflection.ParameterModifier[] modifiers) (at <31c0f51ac5a24a22ba784db24f4ba023>:0)
System.RuntimeType.GetMethodImpl (System.String name, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Reflection.CallingConventions callConv, System.Type[] types, System.Reflection.ParameterModifier[] modifiers) (at <31c0f51ac5a24a22ba784db24f4ba023>:0)
System.Type.GetMethod (System.String name, System.Reflection.BindingFlags bindingAttr) (at <31c0f51ac5a24a22ba784db24f4ba023>:0)
HarmonyLib.AccessTools.DeclaredMethod (System.Type type, System.String name, System.Type[] parameters, System.Type[] generics) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchTools.GetOriginalMethod (HarmonyLib.HarmonyMethod attr) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
Rethrow as HarmonyException: Ambiguous match for HarmonyMethod[(class=Timberborn.MapSystem.MapRepository, methodname=CustomMapNameToFileName, type=Normal, args=undefined)]
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.CreateAndPatchAll (System.Reflection.Assembly assembly, System.String harmonyInstanceId) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
TBMoreMapsDirectories.MoreMapsDirectoriesPlugin.Awake () (at <a4802ee27f00483e9eaaae70a3e104c3>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()

[Info   :   BepInEx] Loading [Quadruple Terrainheight 1.0.2]
[Message:   BepInEx] Chainloader startup complete
[Info   : Unity Log] Starting game version 0.1.1.1-db764df-sw
[Info   : Unity Log] System info:
System: Windows 10  (10.0.19043) 64bit
CPU: AMD Ryzen 9 3950X 16-Core Processor
GPU: NVIDIA GeForce RTX 3090
GPU memory: 24348MB
RAM: 65456MB
Drives:
  C:\ 290.68GB free from 1861.38GB
  D:\ 1096.28GB free from 1862.98GB
  G:\ 1041.47GB free from 1862.98GB
  X:\ 24581.33GB free from 155075.76GB

[Info   : Unity Log] Modded: true, BepInEx
[Info   :   Console] Loading scene: Global, prefixes in scene: timberApi
[Info   :   Console] Loading scene: MainMenu, prefixes in scene:
[Info   : Unity Log] Initialized configurators for MainMenu
[Info   : Unity Log] Initialized configurators for Global
Setting breakpad minidump AppID = 1062090
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561197964475329 [API loaded no]
[Info   : Unity Log] Successfully connected to the Steam client.
```

**Comment by mhgar** [comment]
Hi. The issue appears to be with TimberAPI updating from 0.1.1 to 0.2.0. A fix will be available soon.

**Comment by mhgar** [comment]
Fixed yesterday! Updated to 0.2.1.
